onnxruntime/src/error.rs

```diff
@@ -184,3 +190,10 @@ pub(crate) fn status_to_result(
     let status_wrapper: OrtStatusWrapper = status.into();
     status_wrapper.into()
 }
+
+/// A wrapper around a function on OrtApi that maps the status code into [OrtApiError]
+pub(crate) unsafe fn call_ort<F: FnMut(sys::OrtApi) -> *const sys::OrtStatus>(
+    mut block: F,
+) -> std::result::Result<(), OrtApiError> {
```

Comment:
I had to double-parse the function signature... 😱 

I wanted to comment on trait bound checks appearing in a `where` clause (instead of inside the generic) but decided not to because it's hard to enforce this convention. But looking at the function signature I think something like this would be more clear:
```suggestion
pub(crate) unsafe fn call_ort<F>(mut block: F) -> std::result::Result<(), OrtApiError>
where
    F: FnMut(sys::OrtApi) -> *const sys::OrtStatus,
{
```

Would you mind changing other trait bounds to this format as well?

Reply:
Sure

Reply:
I only found one other instance; fixed